website/docs/r/configuration_entry.markdown

```diff
@@ -0,0 +1,71 @@
+---
+layout: "consul"
+page_title: "Consul: configuration_entry"
+sidebar_current: "docs-consul-resource-configuration-entry"
+description: |-
+  Registers a configuration entry in Consul.
+---
+
+# consul_configuration_entry
```

Comment:
can we name it `consul_config_entry`? We tend to try to use the full word "configuration" in documentation etc as a style thing, but we abbreviated this name in all of the places that use it as an identifier for this resource type. For example our API is `/v1/config/...` and the CLI command is `consul config ...`